src/components/IconBadge/IconBadge.types.ts

```diff
@@ -0,0 +1,9 @@
+import React from 'react'
+
+export type Props = {
+  className?: string
+  icon?: string
```

Comment:
Icon seems to be a predefined set of icons, can we type it with an enum or a list of strings?

Reply:
yup, done!